db/write_batch.cc

```diff
@@ -2010,4 +1820,207 @@ size_t WriteBatchInternal::AppendedByteSize(size_t leftByteSize,
   }
 }
 
+Status WriteBatchInternal::Iterate(const WriteBatch* wb,
```

Comment:
If we put this function right below WriteBatch::Iterate observing the changes would be easier as "git diff" would only highly the changes to the body of the function.